### PR TITLE
DEV: Update scikit-image Version

### PR DESCRIPTION
**Pull request by Srijay-lab**
- Update scikit image version to 0.19.1

### CODE REVIEW

File: requirements.win64.conda.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ dependencies:
   - pytorch>=1.9.0
   - pyyaml>=5.1
   - requests
-  - scikit-image==0.18.3
+  - scikit-image==0.19.1
```
Comment (shaneahmed) [inline requirements.win64.conda.yml:34]:
```suggestion
  - scikit-image>=0.19.1
```

We need to test with `>=19.1`

**Comment by John-P** [comment]
Could we add a comment next to this line to explain why this has to be pinned. This way, when we come back in future, we will know why it is piined.

**Comment by John-P** [comment]
We should update all of the requirements files if possible to keep the versions consistent.

**Comment by John-P** [comment]
Currently investigating why the scikit-image update changes the `phase_cross_correlation` and breaks tests.

**Comment by shaneahmed** [comment]
@Srijay-lab  please also update `setup.py`

**Comment by shaneahmed** [comment]
Please check this https://scikit-image.org/docs/stable/release_notes.html#api-changes

**Review by shaneahmed — APPROVED** [review]
Thanks @Srijay-lab

**Pull request merged**